project.clj

```diff
@@ -7,8 +7,10 @@
                  [cheshire "5.7.1"]
                  [ring/ring-core "1.6.2"]]
   :plugins [[codox "0.8.13"]]
-  :aliases {"test-all" ["with-profile" "default:+1.6:+1.7:+1.8" "test"]}
+  :aliases {"test-all" ["with-profile" "default:+1.6:+1.7:+1.8:+1.9:+1.10" "test"]}
```

Comment:
These changes need to be separated out into their own PR, as they're not directly related to adding stream support.

Reply:
removed and will resubmit separately